src/doc/rustc-dev-guide/src/tests/compiletest.md

```diff
@@ -415,6 +415,10 @@ Compiletest directives like `//@ only-<target>` or `//@ ignore-<target>` are
 supported in `rmake.rs`, like in UI tests. However, revisions or building
 auxiliary via directives are not currently supported.
 
+`rmake.rs` and `run-make-support` may *not* use any nightly/unstable features,
+as they must be compilable by a stage 0 rustc that may be a beta or even stable
+rustc.
```

Comment:
Remark: adjusted this to also say that `run-make-support` may not depend on nightly/unstable features.